helm/aws-operator-chart/Chart.yaml

```diff
@@ -1,2 +1,2 @@
 name: aws-operator-chart
-version: 1.0.0-[[ .SHA ]]
+version: [[ .Version ]]-[[ .SHA ]]
```

Comment:
Should architect have support for `.Version` first? Isn't the corresponding place here: https://github.com/giantswarm/architect/blob/master/template/helm.go#L46-L56

Reply:
Ah, nevermind, just noticed that you wrote this in description.